ts/a11y/SpeechMenu.ts

```diff
@@ -0,0 +1,209 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018-2022 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * @fileoverview Clearspeak preference menu.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import { SpeechExplorer } from './explorer/KeyExplorer.js';
+import { ExplorerMathItem } from './explorer.js';
```

Comment:
Traditionally, we have not included spaces in the braces.  Is this a new style that you want to start using?  Personally, I don't care for it, but I know a lot of people do it,.

Reply:
I think it is `prettier` which adds those automatically. Those spaces are probably from an experimental implementation in SRE. I don't add them manually.
We can make a decision once we decided on linting/formatting for v4.